src/sdk/namespace_fs.js

```diff
@@ -3133,6 +3157,22 @@ class NamespaceFS {
         }
     }
 
+    /**
+     * _check_version_moved recieves key and version_id and checks if the version still exists in one of the optional locations
+     * latest version location or .versions
+     * @param {import('./nb').NativeFSContext} fs_context 
+     * @param {string} key 
+     * @param {string} version_id 
+     */
+    async _check_version_moved(fs_context, key, version_id) {
+        const latest_version_path = this._get_file_path({ key });
+        const verisoned_path = this._get_version_path(key, version_id);
```

Comment:
A typo in the variables names: `verisoned_path` -> `versioned_path` (also `verisoned_path_info`).